cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -300,8 +299,50 @@ export default {
     getCredentialType: function (vc) {
       return getCredentialType(vc.credentialType);
     },
-    getCredentialDisplayData: function (vc, manifestCredential) {
-      return getCredentialDisplayData(vc, manifestCredential);
+    getCredentialDisplayData: function (vc) {
+      return {
+        id: 'aHR0cHM6Ly9pc3N1ZXIub2lkcC51c2Npcy5nb3YvY3JlZGVudGlhbHMvODM2Mjc0NjU',
+        brandColor: 'boatBlue',
+        issuanceDate: '2019-12-03T12:19:52Z',
+        title: 'Permanent Resident Card',
+        icon: 'credential--uscis-icon.svg',
+        properties: [
+          {
+            label: 'Photo',
+            value: '',
+            type: 'image',
+            format: '',
+          },
+          {
+            label: 'Given Name',
+            value: 'JOHN',
+          },
+          {
+            label: 'Family Name',
+            value: 'SMITH',
+          },
+          {
+            label: 'Gender',
+            value: 'Male',
+          },
+          {
+            label: 'Date of Birth',
+            value: '1958-07-17',
+            type: 'date',
+            format: 'yyyy-mm-dd',
+          },
+          {
+            label: 'Country of Birth',
+            value: 'Bahamas',
+          },
+          {
+            label: 'Resident Since',
+            value: '2015-01-01',
+            type: 'date',
+            format: 'yyyy-mm-dd',
+          },
+        ],
+      };
     },
```

Comment:
Why is this hardcoded ? Can't we get details from the API ? Manifest is already stored as part of issuance/save flow. cc: @birtony

Reply:
Not hardcoding anymore with the new commit